### PR TITLE
fix webpack <4 + svelte3 compilation

### PR DESCRIPTION
**Pull request by 0gust1**
**Edit : obsolete comment**
detect webpack version (<4) and set correct configuration if the loader is using the v3 compiler

### CODE REVIEW

**Comment by 0gust1** [comment]
The CI doesn't pass because of the missing webpack dependency (as I had to detect the webpack version). Should I add webpack ^3.0.0 to the peerDependencies of svelte-loader ?

**Comment by 0gust1** [comment]
I don't understand why the CI hang on the webpack dependency (declared as peer) and not on the svelte dependency (also declared as peer).

Can I have some guidance @Rich-Harris ?

File: package.json [hunk 1]
```diff
@@ -28,7 +28,8 @@
     "svelte": "^3.0.0-beta.5"
   },
   "peerDependencies": {
-    "svelte": ">1.44.0"
+    "svelte": ">1.44.0",
+    "webpack": "^3.0.0 || ^4.0.0"
```
Comment (0gust1) [inline package.json]: We maybe also write `"webpack": ">3.0.0"`?

Comment (0gust1) [reply package.json]: Yes, will have to change that, as in #88 someone seems to have the problem with a webpack 2.7 version

**Comment by Gotterbild** [comment]
Looks like this one fixes #88 as well.

**Comment by 0gust1** [comment]
@mrkishi commits squashed and branch push-forced. Tell me if it's good.

**Pull request merged**